### ISSUE TITLE
Can't auto import std::mem::transmute

### ISSUE THREAD

**Issue by chinedufn** [rust-analyzer/rust-analyzer#4883]
Only suggests `std::instrinsics::transmute`

![image](https://user-images.githubusercontent.com/2099811/84606399-df5b5c00-ae73-11ea-8e2c-465900366e94.png)

**Comment by bjorn3** [comment]
`std::mem::transmute` is a re-export of `std::intrinsics::transmute`.

**Comment by chinedufn** [comment]
Ah, well then

**Comment by bjorn3** [comment]
It would be nice to import `std::mem::transmute` instead of `std::intrinsics::transmute` as the later is only available on nightly.

**Comment by flodiebold** [comment]
Yeah, we need to track stability in general (also for completions, for example).

**Comment by jonas-schievink** [comment]
One way to paper over this would be to prefer shorter paths in the `ImportMap` by not just comparing number-of-segments, but also length-of-segments when the compared paths have the same number of segments.

Doesn't fix the actual issue of tracking stability, but it's probably something we want to do anyways.

**Comment by flodiebold** [comment]
I don't know, length of path segments doesn't seem that good a criterion to me; also this only works by luck in this case...

**Comment by ChayimFriedman2** [comment]
Works now.
